src/emmetHelper.ts

```diff
@@ -626,17 +626,20 @@ function applyVendorPrefixes(expandedProperty: string, vendors: string, preferen
 	return prefixedProperty + expandedProperty;
 }
 
+export function parseAbbreviation(abbreviation: string, options: ExpandOptions): any {
+	return parse(abbreviation, options);
+}
 
 /**
  * Expands given abbreviation using given options
  * @param abbreviation string
```

Comment:
this needs to be adjusted, doesn't it?

Reply:
you mean formatted?

Reply:
No, the type isn't `string` anymore.

Reply:
The parse function in the expand module is expecting an abbreviation of type `string`.
The idea is that, having an abbreviation that is a `string`, you can either:
- Expand it as usual, or
- Parse it into a tree structure with this method, then read/modify it, and finally expand it